.github/workflows/ui-tests.yml

```diff
@@ -130,15 +133,26 @@ jobs:
         run: cd ~/frappe-bench/ && bench --site test_site execute erpnext_ui_tests.test_utils.site_setup.execute
 
       - name: cypress pre-requisites
-        run: cd ~/frappe-bench/apps/frappe && yarn add cypress-file-upload@^5 @4tw/cypress-drag-drop@^2 cypress-real-events @testing-library/cypress@^8 --no-lockfile
+        run: |
+          cd ~/frappe-bench/apps/frappe
+          yarn add cypress-file-upload@^5 @4tw/cypress-drag-drop@^2 cypress-real-events @testing-library/cypress@^8 --no-lockfile
 
       - name: Build Assets
         run: cd ~/frappe-bench/ && bench build
         env:
           CI: Yes
 
       - name: UI Tests
-        run: cd ~/frappe-bench/ && bench --site test_site run-ui-tests erpnext_ui_tests --with-coverage --headless
+        run: >
+          cd ~/frappe-bench/
+
+          bench --site test_site run-ui-tests erpnext_ui_tests
+          --with-coverage
+          --headless
+          --ci-build-id $GITHUB_RUN_ID-$GITHUB_RUN_ATTEMPT
+          --
+          --config-file cypress-${{ matrix.container }}.json
+          --group erpnext-${{ matrix.container }}
```

Comment:
This is the main change. Basically on each container, we are running with a different config and hence we can manually split our tests based on files named `cypress-#.json`